math/unpack.wesl

```diff
@@ -44,7 +44,7 @@ fn unpack256(v: vec3) -> f32  {
     return dot(v, f) / 16581375.0;
 }
 
-fn unpack(v: vec3f, base: f32) -> f32 {
+fn unpackBase(v: vec3f, base: f32) -> f32 {
```

Comment:
I hope WESL will support soon overloads so we don't have to deal with this anymore

Reply:
I'm so looking forward to it